sopel/cli/run.py

```diff
@@ -168,14 +168,17 @@ def build_parser():
     """Build an ``argparse.ArgumentParser`` for the bot"""
     parser = argparse.ArgumentParser(description='Sopel IRC Bot',
                                      usage='%(prog)s [options]')
+    add_legacy_options(parser)
+    utils.add_common_arguments(parser)
+
     subparsers = parser.add_subparsers(
         title='sub-commands',
         description='List of Sopel\'s sub-commands',
-        dest='action')
+        dest='action',
+        metavar='{start,configure,stop,restart}')
```

Comment:
Will anyone necessarily remember to undo this `metavar` when `legacy` is removed, or remember to update it when adding new CLI actions?

I'm guessing it's the only not-awful way to hide "legacy" from the action listing in help, though, so this isn't a change request per se.

Reply:
Yeah, it's annoying, but there is a bug listed for that: https://bugs.python.org/issue22848

Reply:
Not encouraging that that bug has been open for over four years… I guess we're stuck with this way.